data/json/monster_factions.json

```
@@ -375,27 +375,40 @@
     "name": "ant",
     "base_faction": "insect",
     "by_mood": [ "zombie", "fungus" ],
-    "neutral": [ "aphid" ]
+    "neutral": [ "aphid", "stag_beetle_larva" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "acid_ant",
     "base_faction": "insect",
     "by_mood": [ "zombie", "fungus" ],
-    "neutral": [ "aphid" ]
+    "neutral": [ "aphid", "stag_beetle_larva" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "aphid",
     "base_faction": "insect",
-    "neutral": [ "ant", "acid_ant" ]
+    "neutral": [ "ant", "acid_ant", "stag_beetle_larva" ]
   },
   {
     "type": "MONSTER_FACTION",
     "name": "crayfish",
     "base_faction": "insect",
     "hate": "fish"
   },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "stag_beetle",
+    "base_faction": "insect",
+    "friendly": "stag_beetle_larva"
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "stag_beetle_larva",
+    "base_faction": "insect",
+    "neutral": [ "ant", "acid_ant" ],
+    "friendly": "stag_beetle"
```

```suggestion
    "base_faction": "insect",
    "by_mood": "zombie"
  },
  {
    "type": "MONSTER_FACTION",
    "name": "stag_beetle_larva",
    "base_faction": "stag_beetle",
    "neutral": [ "ant", "acid_ant" ]
```
As long as the faction is friendly to itself, it should be friendly to a derived faction, and vice-versa. Are these in a social group (or, maybe, do they eat corpses)? If not, they likely should be neutral to zombies and vice-versa.

---

Thanks for the advice!  I think they should be neutral to zombies, as they are largely herbivorous- though there is a lack of research on if real life stag beetles will eat decaying meat, as they love decaying leaves and wood, so some suggest they may be partial saprovores